.bazelrc

```diff
@@ -4,6 +4,8 @@
 
 build --incompatible_strict_action_env
 build --nolegacy_external_runfiles
+build --symlink_prefix=/
```

Comment:
why? it seems like it belongs in your personal bazelrc

Reply:
`bats -r .` recurses into bazel-bin causing infinite loops.  this belongs in e2e/auth actually. will fix it

Reply:
oh, it's a symlink. can't do that. I'll remove this and put into .bazelrc.user

Reply:
done.